custom/icds/forms.py

```diff
@@ -54,6 +54,16 @@ class HostedCCZForm(forms.Form):
                                  required=False, widget=Select(choices=[]))
     file_name = forms.CharField(label=ugettext_lazy("CCZ File Name"), required=False)
     note = forms.CharField(required=False, widget=forms.Textarea(attrs={'rows': 3, 'cols': 15}))
+    status = forms.ChoiceField(label=ugettext_lazy("Status"),
+                               choices=(
+                                   ('', 'Select Status'),
```

Comment:
Not blocking, but can you use an actual label or placeholder for this? It always strikes me as weird to see a placeholder as an actual piece of data.

Reply:
Can you share an example of how that would be?

Reply:
Offlined, fine to use the blank option. Without it, the first option is pre-selected.